### DISCUSSION TITLE
Thumbnails not showing

### DISCUSSION THREAD

**Discussion by scouech** [directus/directus#5515] [Archive]
hey! Me, again!

I've duplicated a directus project to a new server. Everything works well except thumbnails. 
I get a missing image (see 1st screenshot) and when trying to access the thumbnail directly it I get an error (see 2nd screenshot).

The assets are present in the uploads folder. 

When copying the URL to the thumbnail I get
`https://DOMAIN/assets/133e2772-c858-4141-883a-53dd00c43a8a?cache-buster=2021-04-27T11:36:35Z&key=system-large-contain&access_token=TOKENHERE`

Is there a way to "refresh" the thumbnails? What would be the best approach to fix this? 

Cheers!

<img width="868" alt="Screenshot 2021-05-06 at 1 40 55 PM" src="https://user-images.githubusercontent.com/26603052/117292970-1ca45000-ae71-11eb-8149-f4b7a201de12.png">

<img width="1416" alt="Screenshot 2021-05-06 at 1 40 47 PM" src="https://user-images.githubusercontent.com/26603052/117292978-21690400-ae71-11eb-90b8-169266ad3098.png">

**Comment by benhaynes** [comment]
To regenerate thumbnails, all you should have to do is delete them from the storage. They are generated based on their existence. However this could be a different issue with the generation itself... hard to say without research.

**Comment by rijkvanzanten** [comment]
That error EACCESS: permission denied seems to come from the actual storage adapter, being local or otherwise, so make sure that your S3/Azure/GCS user has the correct read permissions, and/or make sure that the node process can access the file on the local filesystem

**Reply by scouech** [reply]
thanks @benhaynes and @rijkvanzanten ! Good to know about the thumbnails from storage!  @rijkvanzanten you were right about the permissions.. I created manually the `uploads` folder when moving things around so the folder didn't have the right permissions. 
To fix it here is what I did:
1. Deleted the `uploads` folder
2. Went to the admin and uploaded a random file 
3. Directus created the folder correctly
4. sftp the content back into the uploads folder

Thanks both, I should have thought of that!

**Reply by rijkvanzanten** [reply]
> Thanks both, I should have thought of that!

Nahh! These are the sorts of issues that you run into once, and then remember forever 😄 The only reason I knew where to look is that I've spent at least a day of my life trying to figure out where `EACCESS: permission denied` comes from 😅
